### ISSUE TITLE
`cancel()` doesn't cancel the job with GDP server

### ISSUE THREAD

**Issue by ashander** [USGS-R/geoknife#411]
Is this intended behavior? I see `cancel` only changes the webprocess object (removes the url from slot `@id` https://github.com/USGS-R/geoknife/blob/b5088ce11ddd2a4a358888fc3b9bd8aab3eb52df/R/cancel-geojob.R#L35) but possibly doesn't cancel the job on the server.

For example, with the code below the request is still processed by GDP. 

```r
job <- geoknife(stencil = c(-89,42), fabric = 'prism'); cancel(job)
```

**Comment by jread-usgs** [comment]
Nice detective work, @ashander :)

You are correct. `cancel` does not actually cancel an active job on the processing server. It was stubbed out with the hopes that a future service that would allow you to cancel _your_ job (but not someone else's) would be supported. 

This function exists because if we don't track active jobs internally in the package it was too easy for people to spin up a ton of accidental processing jobs that would lock up the queue. 

So to answer your question, yes this is the intended behavior. @dblodgett-usgs do you see this function or job tracking/cancelling changing in the future or plan on keeping this as-is?

**Comment by dblodgett-usgs** [comment]
Yes, nice detective work. All it does is cancel the local job state! 

We are getting ready to implement a new back end for the GDP (next year or two). The new back end should support "cancel". So I think we will leave this as is until that's in place.
